app/views/posts/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>All Users</title>
+  <%= stylesheet_link_tag "style" %>
+</head>
+<body>
+  <div class="post-index">
+    <% @author_posts.each do |post| %>
+      <a class="card-link" href="<%= user_post_path(post.id) %>">
+        <div class="post-card">
+          <img src="https://your-image-url.com/image.jpg" alt="Image Description">
+          <div class="content">
```

Comment:
- [ ] As mentioned by the previous :point_up: reviewer kindly use the `recent_comments` method that you created in the `app/models/post.rb` file to render the recent comments in the `Posts#Index` page but still you didn't add `recent_comments` method in `app/models/post.rb` file kindly fix this issue as well :slightly_smiling_face:

Reply:
- Resolved 👍🏽